trade-mapping-to-ipaffs/src/test/java/uk/gov/defra/stw/mapping/toipaffs/common/DepartureTimeMapperTest.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.defra.stw.mapping.toipaffs.common;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import java.time.LocalTime;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import uk.gov.defra.stw.mapping.dto.CodeType;
+import uk.gov.defra.stw.mapping.dto.SpsCertificate;
+import uk.gov.defra.stw.mapping.dto.SpsExchangedDocument;
+import uk.gov.defra.stw.mapping.dto.SpsNoteType;
+import uk.gov.defra.stw.mapping.dto.TextType;
+import uk.gov.defra.stw.mapping.toipaffs.common.utils.DateTimeUtil;
+import uk.gov.defra.stw.mapping.toipaffs.exceptions.NotificationMapperException;
+
+public class DepartureTimeMapperTest {
+  @Mock
+  private DateTimeUtil dateTimeUtil;
+  @Mock
+  private DepartureTimeMapper departureTimeMapper;
+  @Mock
+  private SpsCertificate spsCertificate;
+
+  @Test
+  void map_ArrivalTime() throws NotificationMapperException {
```

Comment:
Same comments here as for ArrivalDateMapperTest

Reply:
fixed